Specification/5._Application_level_conventions.md

```diff
@@ -8,33 +8,31 @@ permalink: /Specification/5._Application_level_conventions/
 
 ### Message type ID
 
-As specified in the CAN bus transport layer specification, the valid range for message type ID is from 0 to 65535,
-inclusive.
+The valid range for *message type ID* is from 0 to 65535, inclusive (specified in [CAN bus transport layer](/Specification/4._CAN_bus_transport_layer)).
 The range is segmented as follows:
 
 ID range        | Purpose
 ----------------|--------------------------------------------------------------
-[0, 20000)      | Standard message types
-[20000, 21000)  | Vendor-specific message types
-[21000, 65536)  | Reserved for future use
+[0, 20000]      | Standard message types
```

Comment:
FYI The open/close brackets were of different types - and that looks weird. I assume there was no reason for this.

Reply:
It was very much intentional. It's a standard mathematical notation that denotes open and closed ranges. A square bracket means that the value is included in the range, a round brace means that the value is not included.

Reply:
Thank you. I will update.